commons/params.py

```diff
@@ -125,3 +125,10 @@
 PROV_SKIP_TEST_FILES_HEALTH_CHECK_PREFIX = ['test_prov', 'test_failure_domain',
                                             'test_multiple_config_deploy', 'test_cont_deployment',
                                             "test_di_deployment"]
+
+# Ceph s3-tests Runner Params
+S3TESTS_DIR = "s3-tests"
+S3TESTS_CONF_ENV = "S3TEST_CONF"
+S3TESTS_CONF_FILE = "s3tests.conf"
+REPORTS_DIR = "reports"
+VIRTUALENV_DIR = "virtualenv"
```

Comment:
why would you need to specify virtualenv in Params?  It is responsibility of another script to take care setting the py env and pythonpath.

Reply:
Using this to get the path to nosetests command. The Makefile executes the ceph/s3-tests bootstrap script which takes care of setting this virtual environment up

Reply:
Let it assume project root to setup virtual env .  Also does it setup virenv every time. There must be an option ruse the older virenv as well.

Reply:
Yes, agreed but here we cleanup the repo for each execution so this s3-tests repo clone is temporary in the local workspace.